frontend/language/src/nb.json

```diff
@@ -247,6 +247,11 @@
   "expression.errorListHeader": "Uttrykket kan ikke lagres på grunn av følgende feil:",
   "expression.false": "Usann",
   "expression.firstOperand": "Første operand",
+  "expression.gatewayActionContext.confirm": "Bekreft",
+  "expression.gatewayActionContext.pay": "Pay",
```

Comment:
As the line above is in Norwegian, maybe line 251, 252, 253, and 254 should be it too?